### DISCUSSION TITLE
The ability to install plugins/addons for some games right from the panel

### DISCUSSION THREAD

**Discussion by BlueFireRO** [pelican-dev/panel#75] [Ideas]
This would add a Plugins/Addons tab that depending on the game would list plugins/addons to install with only 1 click distance,for example for minecraft would list plugins from spigotmc or a similar website

**Comment by SambhavKamboj** [comment]
I mentioned this in the survey too.
Would love to have it.

**Comment by TekExplorer** [comment]
This is definitely going to be a plugin. that is, someone is going to make one.
for minecraft, see the following:
- curseforge (iffy. not sure what the requirements are on their api anymore. i think its overwolf now?)
- spogotmc (doesnt have a real api, but spiget exists)
- papermc's hangar
- modrinth
- polymart

curse and modrinth also include mods (as an example)

**Comment by lancepioch** [comment]
Thanks, this is planned!

**Reply by TekExplorer** [reply]
Shouldn't the discussion remain open until its implemented?
This would be a good place to discuss the "how"
